index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>html and css cards quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+  <style type="quiz.css" media="screen">
+
+  </style>
+</head>
+<body>
+
+<article id="cardOne">
+    <header id="header" class="dbg">
+       <h1>Grey Marble</h1>
+    </header><!-- /header -->
+
+  <div>
```

Comment:
All these divs should be indented to the same level as the header tag above, same for the divs in all the other cards.